packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -145,3 +146,11 @@ export function selfcareIdConflict({
     title: "Selfcare id conflict",
   });
 }
+
+export function tenantIsNotCertifier(tenantId: TenantId): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Organization ${tenantId} not allowed to assign attributes`,
```

Comment:
I know that this is the string in Scala code, but  may I ask you to change it to
```suggestion
    detail: `Organization ${tenantId} not a certifier`,
```
?
this way is more aligned with the error name